source/user-manual/capabilities/osquery.rst

```diff
@@ -18,7 +17,7 @@ How it works
 ------------
 Osquery can be used to expose an operating system as a high-performance relational database. This allows you to write SQL-based queries to explore operating system data.
 
-Bellow you can see some examples of the queries we can make:
+Bellow you can see some examples of the queries you can make:
```

Comment:
Please correct misspelled `Bellow` to `Below`.

Reply:
Sure @pillarsdotnet , thanks! 

I'm going to fix it right now.

Regards!